src/main/resources/static/app/request/request-contents.html

```diff
@@ -313,6 +282,40 @@ <h4 class="h4">
     </div>
 
 
+    <!-- Billing address -->
+    <h4 class="h4">
+        {{ 'Billing address' | translate }}
```

Comment:
I think `Billing details` is a better name here, only the first line has billing address.

Reply:
it is changed now